lib/std/os/windows/ntdll.zig

```diff
@@ -375,3 +379,53 @@ pub extern "ntdll" fn NtFreeVirtualMemory(
     RegionSize: *SIZE_T,
     FreeType: ULONG,
 ) callconv(.winapi) NTSTATUS;
+
+pub extern "ntdll" fn RtlAddVectoredExceptionHandler(
+    First: ULONG,
+    Handler: ?VECTORED_EXCEPTION_HANDLER,
+) callconv(.winapi) ?LPVOID;
+pub extern "ntdll" fn RtlRemoveVectoredExceptionHandler(
+    Handle: HANDLE,
+) callconv(.winapi) ULONG;
+
+pub extern "ntdll" fn RtlInitializeCriticalSection(
+    lpCriticalSection: *CRITICAL_SECTION,
+) callconv(.winapi) void;
+pub extern "ntdll" fn RtlEnterCriticalSection(
+    lpCriticalSection: *CRITICAL_SECTION,
+) callconv(.winapi) void;
+pub extern "ntdll" fn RtlLeaveCriticalSection(
+    lpCriticalSection: *CRITICAL_SECTION,
+) callconv(.winapi) void;
+pub extern "ntdll" fn RtlDeleteCriticalSection(
+    lpCriticalSection: *CRITICAL_SECTION,
+) callconv(.winapi) void;
```

Comment:
According to [the System Informer/Process Hacker headers](https://github.com/winsiderss/systeminformer/tree/master/phnt), these `CriticalSection` functions all return `NTSTATUS`.

(alternatively, these functions can just be deleted, as they are no longer used [they were used for a Windows mutex implementation but now the `SRWLock` functions are used for that instead])